### PR TITLE
185: animals, cooking, food and music B1

### PR DESCRIPTION
**Pull request by greenEkatherine**

### CODE REVIEW

File: app/src/main/res/raw/data.json [hunk 1]
```diff
@@ -604,6 +728,17 @@
         "topics": ["животные"]
         }}
     ],
+    [
+        {"EN":{
+            "word": "wool",
+            "topics": ["animals"],
```
Comment (omen273) [inline app/src/main/res/raw/data.json:734]: I think we need to remove this word because of vegans

File: app/src/main/res/raw/data.json [hunk 2]
```diff
@@ -3548,6 +4192,39 @@
...
+        }}
+    ],
+    [
+        {"EN":{
+            "word": "thin",
+            "topics": ["cooking and eating"],
+            "level": "intermediate(B1)"
+        }},
+        {"RU": {
+            "word": "водянистый",
```
Comment (omen273) [inline app/src/main/res/raw/data.json:4213]: тонкий?

**Review by omen273 — CHANGES_REQUESTED** [review]
Please address my comments

File: app/src/main/res/raw/data.json (hunk 2 above)
Comment (greenEkatherine) [reply app/src/main/res/raw/data.json:4213]: discussed offline

Comment (omen273) [reply app/src/main/res/raw/data.json:4213]: Decided to stay as is

File: app/src/main/res/raw/data.json (hunk 1 above)
Comment (greenEkatherine) [reply app/src/main/res/raw/data.json]: removed

**Review by omen273 — APPROVED** [review]

**Pull request merged**